sdk-scheme-adapter/v2_0_0/backend/openapi.yaml

```diff
@@ -133,11 +133,11 @@ paths:
       summary: Requests information relating to a bulk transfer identified by the specified identifier value.
       tags:
         - BulkTransfers
-  /otp/{requestToPayId}:
+  /otp/{transactionRequestId}:
```

Comment:
I just replaced all the requestToPayId occurrences with transactionRequestId to avoid confusion. This change may be for other story. But it's just name of the parameter and I hope it doesn't affect any implementations. And about the additional params, we can handle it in the corresponding story. What do you think?